test/data/user.get.yaml

```diff
@@ -3,4 +3,10 @@ id: 123
 username: batman
 scmContext: github:github.com
 settings:
-    displayJobNameLength: 20
+    1:
```

Comment:
what is this number referring to?

Reply:
1 means pipeline number. It would be like:
```
1: {
  displayJobNameLength: 10,
  showPRJobs: true
},
11: {
  displayJobNameLength: 10,
  showPRJobs: true
},
some_arbitary_vars: 'xyz'
```